### ISSUE TITLE
VM cannot run normally in kvm UEFI LEGACY mode

### ISSUE THREAD

**Issue by div8cn** [apache/cloudstack#4245]
##### ISSUE TYPE

 * Bug Report
##### COMPONENT NAME

~~~
KVM 
UEFI
~~~

##### CLOUDSTACK VERSION

~~~
4.14.0.0
~~~

##### OS / ENVIRONMENT

Hypervisor:  CentOS7.6
Compiled against library: libvirt 4.5.0
Using library: libvirt 4.5.0
Using API: QEMU 4.5.0
Running hypervisor: QEMU 2.12.0

OVMF: OVMF-20180508-6.gitee3198e672e2.el7.noarch

##### SUMMARY

Under KVM Hypervisor
In UEFI LEGACY mode, the VM cannot enter the system normally, and "Guest has not initialized the display (yet)." is displayed in the console.

![image](https://raw.githubusercontent.com/div8cn/image/master/202008/centos7-kvm-uefi-1.png)
VM starts normally in UEFI Secure mode

After comparison, we found that:
**In UEFI LEGACY mode, the VM uses pc-i440f chip,**
**In UEFI Secure mode, the Q35 chip is used.**

After I switched the VM chip from pc-i440fx-rhel7.6.0 to pc-q35-rhel7.6.0 (manually modify the configuration file), the VM can enter the system normally.

**so. UEFI under CentOS7 should run under Q35 chip (UEFI LEGACY AND UEFI SECURE)**

##### STEPS TO REPRODUCE

~~~
1.Deploy VM through template, boot type select UEFI.Boot Mode select LEGACY 
2.After the VM runs successfully, open the console

~~~

##### EXPECTED RESULTS

~~~
The console displays guest OS information, or UEFI shell.
~~~

##### ACTUAL RESULTS

~~~
Guest has not initialized the display (yet).
~~~

**Comment by DaanHoogland** [comment]
@div8cn do you have resources to work on this?
should we set a milestone yet?

**Comment by div8cn** [comment]
I have resources to provide testing
@DaanHoogland

**Comment by DaanHoogland** [comment]
ok, we might have to postpone to 4.16 than as development work will be required i guess

**Comment by rhtyd** [comment]
@DaanHoogland can you triage and advise if this is a bug or config issue, we can ask @Spaceman1984 to look into this

**Comment by rhtyd** [comment]
Fixed in #4576
